addons/project/project.py

```diff
@@ -434,6 +434,10 @@ def create(self, cr, uid, vals, context=None):
         return project_id
 
     def write(self, cr, uid, ids, vals, context=None):
+        for record in self.browse(cr, uid, ids, context=context):
+            if 'active' in vals and not vals.get('active') and ('state' in vals and vals.get('state') in ('open', 'pending', 'template') or 'state' not in vals and record.state in ('open', 'pending', 'template')):
```

Comment:
ok so Running project should not deactivate. can you put python constraint to check this?